packages/langium/src/grammar/grammar-util.ts

```diff
@@ -44,6 +44,10 @@ export function isDataTypeRule(rule: ast.ParserRule): boolean {
     return false;
 }
 
+export function isCommentTerminal(terminalRule: ast.TerminalRule): boolean {
+    return terminalRule.$container.hiddenTokens.some(e => e.ref === terminalRule) && !' '.match(terminalRule.regex);
```

Comment:
Shall we use the regexp visitor to detect whitespace-only terminals similarly to how you did it to detect multiline comments?

Reply:
I think this solution is the best, as it does not rely on our custom detection behavior with the visitor (probably missing a lot of cases).